docs/doc/30-reference/20-functions/90-context-functions/timezone.md

```diff
@@ -2,31 +2,34 @@
 title: TIMEZONE
 ---
 
-Return the timezone for the connection.
+Returns the timezone for the current connection. 
```

Comment:
Timezones is a setting not a context function, please move to setting a chapter :)